pom_osx.xml

```diff
@@ -0,0 +1,212 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.diffblue</groupId>
+  <artifactId>deeptestutils</artifactId>
+  <version>0.0.1</version>
+  <packaging>jar</packaging>
+
+  <name>Deeptest-utils</name>
+  <!-- part of maven central requirements is to provide sufficient metadata   -->
+  <!-- @todo review description -->
+  <description>dependency for diffblue deeptest</description>
+  <!-- @todo review url -->
+  <url>https://github.com/diffblue/deeptest-utils</url> 
+
+  <!-- must declare the license(s) used for distributing your components. -->
+  <!-- @todo review license - apache vs MIT vs... -->
+  <licenses>
+    <license>
+      <name>BSD-4-Clause-License</name>
+      <url>https://spdx.org/licenses/BSD-4-Clause.html</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <!-- In order to be able to associate the project it is required to add a developers section. -->
+  <!-- @todo - review this section -->
+  <developers>
+    <developer>
+      <name>DiffBlue</name>
+      <email>info@diffblue.com</email>
+      <organization>diffblue</organization>
+      <organizationUrl>http://www.diffblue.com</organizationUrl>
+    </developer>
+  </developers>
+  <!-- if using the maven-release plugin: -->
+  <!-- http://central.sonatype.org/pages/apache-maven.html#performing-a-release-deployment-with-the-maven-release-plugin -->
+  <!-- then the connection to source control system is required.  -->
+  <scm>
+    <url>https://github.com/diffblue/deeptest-utils</url>
+    <!-- @todo review this - why needed a 'connection' vs 'developerConnection -->
+    <developerConnection>scm:git:git@github.com:diffblue/deeptest-utils.git</developerConnection>
+  </scm>
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id><!-- nb this must be the same as the 'id' field in 'settings.xml' -->
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <!-- BELOW NOT REQUIRED WHEN USING 'NEXUS staging plugin' -->
```

Comment:
Let's just remove it then? Since we are using Nexus staging plugin?

Reply:
Done.